docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+:::
+
+Now, our contract is a little more populated:
+
+![Contract Wizard SafeMint](intro-to-erc721s/4-wizard-safemint.png)
+
+For this simple example, we will not add any additional functionality to the `safeMint` function. Currently, it mints one NFT to the address specified in the function call. There is no cost to mint the NFT other than the gas fee for the transaction itself.
+
+This `safeMint` function currently doubles as an airdrop function because the address the NFT is minted to does not need to be the function owner. This functionality becomes very useful when NFT collection owners want to give away NFTs for free outside of the normal minting window.
+
+At this point, our smart contract is ready. At the top, you can click `Open in Remix` to get ready to deploy your smart contract.
+
+![Contract Wizard Open Remix](intro-to-erc721s/5-open-remix.png)
+
+## Deploying the Smart Contract with Remix
+
+[Remix IDE](https://remix.ethereum.org/) is a solidity compiler that allows you to edit, compile, and deploy your smart contract. This will prevent you from needing to download any other coding environments at this stage.
+
+Once you've imported your contract, the first thing you need to do is compile it. Hit the `Compile` button on the left-hand side. You could also use the keyboard shortcut `Ctrl / Command + S`.
+
+![Remix Compile](intro-to-erc721s/6-remix-compile.png)
+
+Once completed, you will get a green checkmark on the far left tab and will see options to Publish on Ipfs or Swarm. Those are not important to our tutorial. Next, you will click on the bottom tab on the left-hand side to move to the deployment page.
+
+![Remix Deploy Page](intro-to-erc721s/7-deploy-page.png)
+
+Now, we need to change the environment that Remix will try to use to deploy the smart contract. Click on the `Environment` drop-down, and select `Injected web3`.
+
+![Remix Web3](intro-to-erc721s/8-web3.png)
+
+This should prompt you to connect with your MetaMask account. Once connected, you can verify the correct connection by checking that the Account number matches your MetaMask address.
+
+![Remix MetaMask](intro-to-erc721s/9-metamask.png)
+
+Now click on the `Contract` drop-down and select the contract you created and compiled. It should show up with the name you gave it in the Open Zeppelin Wizard.
+
+![Remix Contract](intro-to-erc721s/10-contract.png)
+
+Now, click deploy. This will open MetaMask and ask you to confirm the transaction. Click `Confirm`.
+
+![MetaMask Accept](intro-to-erc721s/11-accept.png)
+
+It may take a second, but once completed, your newly deployed contract will appear underneath the `Transactions Recorded` field.
+
+![Remix Record](intro-to-erc721s/12-transaction-record.png)
+
+Copy your contract’s address and open the [Snowtrace Testnet Explorer](https://testnet.snowtrace.io/). Paste your contract address in the search bar, and click `Search`.
```

Comment:
In here and following places where testnet.snowtrace.io is used, it would be useful to provide links to the transaction. The pictures were not very clear, there is no way to enlarge the pictures to see the detail.